app/templates/partials/footer.html

```diff
@@ -0,0 +1,58 @@
+<footer class="footer" role="contentinfo">
+    <div class="container">
+        <div class="grid">
+            <div class="grid__col col-4@m">
+                <p class="venus footer__heading"><strong>Legal information</strong></p>
+                <ul class="list list--bare">
+                    <li>
+                        <a href="/cookies-privacy"
```

Comment:
Should this be pointing at: "https://surveys.ons.gov.uk/cookies-privacy"? As that route doesn't exist in RH.

Reply:
@jcox-dev I think the plan is to duplicate the privacy page so RH has it's own social privacy page has it's own  (due to issues with business 'sign in' link etc on https://surveys.ons.gov.uk/cookies-privacy).  (https://trello.com/c/d0U2sz8L/278-sus021-update-to-content-of-surveys-privacy-page-to-include-information-for-social-surveys)

Reply:
@jcox-dev I'll revert these links and leave that work for sus021.